src/app/portlets/shared/dot-content-types-edit/components/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.spec.ts

```diff
@@ -175,6 +182,24 @@ describe('DotContentTypeFieldsVariablesTableRowComponent', () => {
         expect(comp.cancel.emit).toHaveBeenCalledWith(comp.variableIndex);
     });
 
+    it('should disabled save button when new variable key added is duplicated', () => {
+        comp.fieldVariable = { key: 'Key1', value: '' };
+        comp.variablesList = [comp.fieldVariable, ...mockFieldVariables];
+        comp.showEditMenu = true;
```

Comment:
Is there a test where the UI action test that the `showEditMenu` gets set to true?

Reply:
yes, 

https://github.com/dotCMS/core-web/blob/42a895c8b7999bde4e883f8a7b524136701fb990/src/app/portlets/shared/dot-content-types-edit/components/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.spec.ts#L134